hammers/scripts/lease_stack_reaper.py

```diff
@@ -120,24 +120,24 @@ def find_stacked_leases(self, leases):
             if stacked_previous or stacked_next:
                 stacked.append(leases[i])
 
-        return [x for x in stacked if (x[2] - x[1]).days > 1]
+        return [x['id'] for x in stacked if (end_date - start_date).days > 1]
 
-    def print_info(self, leases):
+    def print_info(self, lease_ids):
         """Return dict of info for console output."""
         return {
             'user_name': self.name,
             'user_email': self.email,
-            'leases': leases}
+            'leases': lease_ids}
 
 
-def send_delete_notification(gpu_user, leases, sender):
+def send_delete_notification(gpu_user, leas_ids, sender):
```

Comment:
```suggestion
def send_delete_notification(gpu_user, lease_ids, sender):
```

If you set up a linter in your IDE it will catch things like this. I use pylama at the moment and have it set up to automatically lint on save.

Reply:
Yea, I have one in atom. I just switched back to sublime because it's lighter weight. It seems the combination of zoom, atom and too many tabs open was freezing up my laptop.